lib/js/jsre.nim

```diff
@@ -0,0 +1,42 @@
+## Regular Expressions for the JavaScript target.
+## * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions
+
+type RegExp* {.importjs.} = object    ## Regular Expressions for JavaScript target.
+  flags* {.importjs.}: cstring        ## cstring that contains the flags of the RegExp object.
+  dotAll* {.importjs.}: bool          ## Whether ``.`` matches newlines or not.
+  global* {.importjs.}: bool          ## Whether to test the regular expression against all possible matches in a string, or only against the first.
+  ignoreCase* {.importjs.}: bool      ## Whether to ignore case while attempting a match in a string.
+  multiline* {.importjs.}: bool       ## Whether or not to search in strings across multiple lines.
+  source* {.importjs.}: cstring       ## The text of the pattern.
+  sticky* {.importjs.}: bool          ## Whether or not the search is sticky.
+  unicode* {.importjs.}: bool         ## Whether or not Unicode features are enabled.
+  lastIndex* {.importjs.}: cint       ## The lastIndex is a read/write integer property of regular expression instances that specifies the index at which to start the next match.
+  input* {.importjs.}: cstring        ## The value of the input property is modified whenever the searched string on the regular expression is changed and that string is matching.
+  lastMatch* {.importjs.}: cstring    ## The value of the lastMatch property is read-only and modified whenever a successful match is made.
```

Comment:
less verbose = more readable. replace with:
```
lastMatch* {.importjs.}: cstring  ## read-only and modified on successful match
lastParen* {.importjs.}: cstring  ## ditto
```

(and i actually mean litteral ditto, not copying the comment `read-only and modified on successful match`)